fab/fabfile.py

```diff
@@ -143,8 +143,8 @@ def tag_commit(self):
             sha=self.deploy_ref,
             obj_type='commit',
             tagger={
-                'name': user.name,
-                'email': user.email,
+                'name': user.login,
```

Comment:
i noticed some people don't have a name: https://github.com/gcapalbo. and this hardly matters cause you can't even see who made the tag in the UI